cmd/ssm-session/main.go

```diff
@@ -358,13 +361,13 @@ func addInstanceToTmuxWindow(tmuxWindow *gomux.Window, profile string, region st
 	return tPane.Exec(fmt.Sprintf("aws ssm start-session --profile %s --region %s --target %s", profile, region, instanceID))
 }
 
-func startSelectionPrompt(instances *instance.InstanceInfoSafe, tags ssmhelpers.ListSlice) (selectedInstances []instance.InstanceInfo, err error) {
+func startSelectionPrompt(instances *instance.InstanceInfoSafe, totalInstances int, tags ssmhelpers.ListSlice) (selectedInstances []instance.InstanceInfo, err error) {
 	instanceIDList := []string{}
 	promptList := instances.FormatStringSlice([]string(tags)...)
 	fmt.Println("      ", promptList[0])
 
 	prompt := &survey.MultiSelect{
-		Message: "Select the instances to which you want to connect:",
+		Message: "Showing " + strconv.Itoa(len(instances.AllInstances)) + "/" + strconv.Itoa(totalInstances) + " instances. Make a selection:",
```

Comment:
Can we do this with sprintf instead to make it cleaner?

`fmt.Sprintf("Showing %d/%d instances. Make a selection:", len(instances.AllInstances), totalInstances)`

Reply:
Good call :+1: